src/DIG/Dialog.jsx

```diff
@@ -22,7 +22,7 @@ let {
 type = type ?? "dialog";
 enableCloseButton = enableCloseButton ?? false;
 
-if (["alert", "dialog"].indexOf(type) < 0) {
+if (["alert", "dialog", "custom"].indexOf(type) < 0) {
```

Comment:
Would it work to use the existing `enableCloseButton` (setting it false) instead of needing to add this `custom` type?

Reply:
It doesn't work. Due to all 'dialog' types getting a close button

Reply:
Oh I see - you're right. That seems like something we should fix. We could still remove the `custom` type and then just update `enableCloseButton` like so:

```
enableCloseButton = enableCloseButton ?? type === 'dialog';
```